lib/schema/resolvers/User.ts

```diff
@@ -1,12 +1,121 @@
-import type { UserResolvers } from "@/schema";
+import { sql } from "@/datasources/postgres";
+import type { Organization, PageInfo, UserResolvers } from "@/schema";
+import { type WithKey, decodeGlobalId } from "@/util";
 
 export const User: UserResolvers = {
-  async authentication_provider(parent, _, ctx) {
-    if (!parent.authentication_provider_id) return null;
-    return ctx.orm.tag.load(parent.authentication_provider_id as string);
+  async authenticationProvider(parent, _, ctx) {
+    if (!parent.authenticationProviderId) return null;
+    return ctx.orm.tag.load(parent.authenticationProviderId as string);
   },
   language(parent, _, ctx) {
-    return ctx.orm.language.byId.load(parent.language_id as string);
+    return ctx.orm.language.byId.load(parent.languageId as string);
+  },
+  async organizations(parent, args, _) {
+    const { first, last } = args;
+    const parentId = decodeGlobalId(parent.id as string).id;
+    const after = args.after ? decodeGlobalId(args.after).id : null;
+    const before = args.before ? decodeGlobalId(args.before).id : null;
+
+    const rows = await sql<WithKey<Organization>[]>`
+      SELECT
+          customeruuid AS _key,
+          encode(('organization:' || customeruuid)::bytea, 'base64') AS id,
+          (customerenddate IS null OR customerenddate > now()) AS active,
+          customerstartdate::text AS "activatedAt",
+          customerenddate::text AS "deactivatedAt",
+          customerexternalid AS "billingId",
+          languagemasteruuid AS "nameId"
+      FROM public.workerinstance
+      INNER JOIN public.customer
+          ON workerinstancecustomerid = customerid
+      INNER JOIN public.languagemaster
+          ON customernamelanguagemasterid = languagemasterid
+      WHERE
+          workerinstanceworkerid = (
+              SELECT workerid
+              FROM public.worker
+              WHERE workeruuid = ${parentId}
+          )
+          ${
+            after
+              ? sql`
+          AND workerinstancecustomerid > (
+              SELECT customerid
+              FROM public.customer
+              WHERE customeruuid = ${after}
+          )
```

Comment:
i might need more coffee to understand this... so we are paginating on the customer's uuid, but then using `>` on the `customerid`... am i just dumb

Reply:
you want a stable sort for pagination, uuids are not stable (at least until we're using v7 everywhere) so you could fairly easily miss a row if you are paginating at the same time as an INSERT